glacier/src/exporters/svg.ts

```diff
@@ -0,0 +1,47 @@
+import * as vl from "vega-lite";
+declare global {
+    interface Element {}
+}
+import * as vega from "vega";
+import redux = require("redux");
+import {ModelState} from "../";
+import {Exporter} from "./";
+
+export function createSvgExporter(store: redux.Store<ModelState>) {
```

Comment:
The data needs to come from Store.  I'm not 100% we want to use a different store it seems kinda coupled to me.